clr-installer/main.go

```diff
@@ -138,6 +139,11 @@ func execute(options args.Args) error {
 		", built on " + model.BuildDate)
 
 	if options.PamSalt != "" {
+		if status, err := user.IsValidPassword(options.PamSalt); !status {
+			fmt.Printf("%s\n", err)
```

Comment:
We should not exit, but instead, return an error
```suggestion
			return err
```

Reply:
okay Thanks. I am going to add a extra carriage return to separate the error from the output of the password. Can be a little hard to see